modules/user_management.js

```diff
@@ -74,6 +74,74 @@ class UserManagement {
         }]
       }
     })
+
+    // Add a single literature quiz point to a user.
+    dispatcher.command('addpoint', this._guild.id, (authorMember, tokenized) => {
+      util.requireRole(authorMember, this._managementRole)
+
+      const USAGE = `${this._config.get('commandPrefix') || 'p!'}addpoint @user`
+      util.requireParameterCount(tokenized, 1, USAGE)
+      util.requireParameterFormat(tokenized[0], arg => arg.startsWith('<@') && arg.endsWith('>'), USAGE)
+
+      const userId = tokenized[0].replace(/[<@!>]/g, '')
+
+      util.log(`addpoint to ${userId} by ${authorMember.id}`)
+      
+      // Call a helper method that conducts the increment and produces the output embed
```

Comment:
Personally, I would just inline this and `_removeLitQuizPoint()`.

Reply:
By inline you mean to remove the helper methods? I really did it this way because it was the easiest way to use an async method and using await that I understood. Pointers appreciated. :D

Reply:
The callback itself can be async:

```js
dispatcher.command('addpoint', this._guild.id, (authorMember, tokenized) => async {
  util.requireRole(authorMember, this._managementRole)
  ...
})
```

then you can `await` on things within the callback.